app/views/appointments/new.html.erb

```diff
@@ -55,7 +55,34 @@
         <div class="appointment">
           <input type="submit" value="Submit">
         </div>
-      </form>
+      </form> -->
+      <%= form_for @appointment do |f| %>
+        <div>
+        <%= f.label :first_name %>
```

Comment:
Why would I need to give you my first/last names and email address if I'm already logged in??